build.sbt

```diff
@@ -905,12 +905,12 @@ lazy val kernelBenchmarks = (project in file("kernel/kernel-benchmarks"))
     ),
   )
 
-lazy val unity = (project in file("unity"))
+lazy val kernelUnityCatalog = (project in file("kernel/unitycatalog"))
```

Comment:
not in this PR but related to this, can we make sure all the kernel modules are consistent in directory structure? 

right now all the other kernelX are in `kernel/kernel-X` where kernel in `kernel-X` is redundant.

Reply:
So:

```
kernel
-- api
-- defaults
-- benchamrks
-- unitycatalog
```

Reply:
yep